### DISCUSSION TITLE
If my pc restarts ?

### DISCUSSION THREAD

**Discussion by syedhuzaifaali660** [swar/Swar-Chia-Plot-Manager#175] [Q&A]
will the plot manager start by it self and my plotting will continue from where it left of ? or will the previous plots which were about to be completed will be deleted ???

**Comment by raid5** [comment]
You can't resuming plotting, so if your pc restarts you have to start the plotting process all over.

**Reply by syedhuzaifaali660** [reply]
Thank you for your reply.
I have one more question my pc got restarted and my m.2 ssd (temp destination) was full the because of that the plot manager was not making new plots so will there be an issue if I empty my m.2 ssd ???

**Reply by raid5** [reply]
If the plotting was interrupted, then yea you'll want to delete all of those temp files before starting plotting again

**Comment by paullepa** [comment]
In my setup, I setup in scheduler a task to "del /q  Q:\my_plot_dir\*.*"  so the system clean up any remnants for me on reboot.
